src/Chapter1.hs

```diff
@@ -568,8 +571,15 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
 
+isVowel :: Char -> Bool
+isVowel 'a' = True
+isVowel 'e' = True
```

Comment:
@vrom9, is there a way to simplify the typing, e.g. as in Kotlin where you list multiple matches separated by comma?